.circleci/config.yml

```diff
@@ -200,12 +200,20 @@ workflows:
                  sonatypeBundleRelease"
           requires:
             - publish_docker_locally
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
These blocks are leftovers of https://github.com/codacy/codacy-analysis-cli/pull/364